src/adapters/postgres/user-adapter.ts

```diff
@@ -23,14 +23,15 @@ import { UserRoleMapping } from "src/rbac/assign-role/entities/assign-role.entit
 import { Tenants } from "src/userTenantMapping/entities/tenant.entity";
 import { Cohort } from "src/cohort/entities/cohort.entity";
 import { Role } from "src/rbac/role/entities/role.entity";
+import { maskMobileNumber, maskEmail, maskDateOfBirth, encrypt } from "src/utils/mask-data";
```

Comment:
@souravbhowmik1999 
Can you add in your tsconfig.json

"paths": {
      "@utils/*": ["src/utils/*"] // Resolve paths as aliases
    },
    
    and use it like
import { maskMobileNumber, maskEmail, maskDateOfBirth, encrypt } from "@utils/mask-data";